src/store/modules/cart.ts

```diff
@@ -11,10 +15,35 @@ const getters = {
   },
   priceTotal(state: CartState): number {
     return state.priceTotal;
+  },
+  getPage(state: CartState): CartItem[] {
+    return state.paginatedItems[state.currentPage - 1] || [];
   }
 };
 
 const mutations = {
+  prevPage: function(state: CartState): void {
+    state.currentPage > 1 ? state.currentPage-- : null;
+  },
+  nextPage: function(state: CartState): void {
+    state.currentPage < state.pagesCount ? state.currentPage++ : null;
+  },
+  setPage: function(state: CartState, page: number): void {
+    state.currentPage = page;
+  },
+  calculatePagesCount: function(state: CartState) {
+    state.pagesCount = Math.ceil(state.items.length / state.itemsPerPage);
+    state.currentPage > state.pagesCount ? (state.currentPage = 1) : null;
+  },
+  setPages: function(state: CartState) {
+    state.paginatedItems = [];
+    const pageSize: number = state.itemsPerPage;
+    const items: CartItem[] = [...state.items];
+    for (let i = 0; i < items.length; i += pageSize) {
```

Comment:
You can also use `.forEach`

Reply:
not really cause I dont want as many iterations as there is items in the array (i += pageSize where pageSize is currently 3 => so for 9 items I do 3 iterations)